bundle/edu.gemini.wdba.xmlrpc.server/src/main/java/edu/gemini/wdba/server/osgi/WDBAServerActivator.java

```diff
@@ -79,9 +87,36 @@ public WdbaContext addingService(ServiceReference<IDBDatabaseService> ref) {
                         }
 
                         final WdbaContext ctx = new WdbaContext(site, new WdbaGlueService(db, user), user);
+
+                        fireService     = FireService.loggingOnly(db);
```

Comment:
For now, it only logs the `FireMessage`s as the are produced.  They will need to be posted to a server in production.